samples/mixed/provider/src/main.rs

```diff
@@ -124,10 +123,87 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
 
     info!("The Provider has started.");
 
-    debug!("Preparing the Provider's DTDL.");
-    let provider_dtdl_path = find_full_path("content/mixed.json")?;
-    let dtdl = retrieve_dtdl(&provider_dtdl_path)?;
-    debug!("Prepared the Provider's DTDL.");
+    let mut telemetry_operations = Vec::new();
+    telemetry_operations.push(String::from("Subscribe"));
+    telemetry_operations.push(String::from("Unsubscribe"));
+
+    let mut property_operations = Vec::new();
+    property_operations.push(String::from("Subscribe"));
+    property_operations.push(String::from("Unsubscribe"));
+    property_operations.push(String::from("Set"));
+
+    let mut command_operations = Vec::new();
+    command_operations.push(String::from("Invoke"));
+
+    // AmbientAirTemperature
```

Comment:
Consider having a function to create EndpointInfo for ambient_air_temperature, air-conditioning, hybrid battery and show notification.

Reply:
I have rewritten the code to make it more condensed.  I think that we can now get away with this function for now.